src/compose/ComposeBox.js

```diff
@@ -302,7 +304,7 @@ class ComposeBox extends PureComponent<Props, State> {
         <View style={styles.composeBox} onLayout={this.handleLayoutChange}>
           <View style={styles.alignBottom}>
             <ComposeMenu
-              narrow={narrow}
+              narrow={this.getDestinationNarrow()}
```

Comment:
Hmm, this feels kind of surprising. Usually we use a `narrow` prop to mean "the narrow the user is looking at". What does `ComposeMenu` do with this prop -- will it do something unexpected because it assumes this is the narrow in view?

Looking at that component, the answer seems to be:
* Indeed it does assume that
* but what it does with that is to go apply some other logic to try to pick a topic to send to!
* so what this change ends up doing is bypassing that logic, because it looks for `isStreamNarrow` and this will no longer ever pass a stream narrow.

Let's avoid confusion here. :-)
* Cut the now-dead `getNarrowToSendTo` over in that code.
* Rename this prop to, say, `narrowToSendTo` (maybe `sendNarrow` if you want to be terse), so it's clear it doesn't have the same meaning as we conventionally give to a `narrow` prop.